node/src/components/consensus/era_supervisor.rs

```diff
@@ -176,6 +179,8 @@ where
             protocol_config.protocol_version.patch as u32,
         );
         let activation_era_id = protocol_config.last_activation_point;
```

Comment:
You can remove that now — you duplicated it two lines below.
```suggestion
```

Reply:
Removed in [`45e2470` (#1068)](https://github.com/CasperLabs/casper-node/pull/1068/commits/45e247049e02d72cbd9c9044cf0e7535f192b9fe)